.github/mergify.yml

```diff
@@ -0,0 +1,61 @@
+queue_rules:
+  - name: default
+    conditions:
+      - "#approved-reviews-by>=1"
+      - base=main
+      - check-success=📚 Documentation
+      - check-success=Schutzbot on GitLab
+      - check-success=Test Suite (test.mod)
+      - check-success=Test Suite (test.run.test_assemblers)
+      - check-success=Test Suite (test.run.test_boot)
+      - check-success=Test Suite (test.run.test_noop)
+      - check-success=Test Suite (test.run.test_sources)
+      - check-success=Test Suite (test.run.test_stages)
+      - check-success=Test Suite (test.src)
+      - check-success=Regenerate Test Data
+      - check-success=Spell check
+      - "check-success=LGTM analysis: Python"
+      - check-success=codecov/project
+      - "check-success~=rpm-build:.*"
+    
+pull_request_rules:
```

Comment:
Can we put both PR rules under one `pull_request_rules` object? See https://docs.mergify.com/examples/#less-strict-rules-for-stable-branches

Reply:
good point